blockstore/splitstore/markset_bloom.go

```diff
@@ -62,6 +62,11 @@ func (s *BloomMarkSet) saltedKey(cid cid.Cid) []byte {
 	key := make([]byte, len(s.salt)+len(hash))
 	n := copy(key, s.salt)
 	copy(key[n:], hash)
+	// FYI, you probably don't need a cryptographic hash function here given that we're salting
+	// anyways.
+	// You could use https://golang.org/pkg/hash/maphash/
+	// (probably)
+	// (if it's a perf issue)
```

Comment:
I'd rather have a crypto hash here, as a collision would totally kill debugability.

Reply:
But yeah, if we have perf problems with bloom filters we can totally change the hash function.